sdk/storage/azure_storage_blob/src/models/blob_properties.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core::headers::Headers;
+
+/// Properties of an Azure Storage blob.
+///
+#[derive(Clone, Default, Debug)]
```

Comment:
Models should derive `SafeDebug` unless you're sure they will *never* have PII: https://azure.github.io/azure-sdk/rust_introduction.html#rust-client-methods-configuration-debug (this says "DO" but that's a mistake; it should be "DO NOT" and I'll fix).